StudioCore/MsbEditor/ModelEditorScreen.cs

```diff
@@ -68,10 +68,6 @@ public void Update(float dt)
                     var dim = box.GetDimensions();
                     var mindim = Math.Min(dim.X, Math.Min(dim.Y, dim.Z));
                     var maxdim = Math.Max(dim.X, Math.Max(dim.Y, dim.Z));
-                    var basespeed = (float)Math.Sqrt(mindim / 3.0f);
```

Comment:
We can revisit this later, but I'd prefer not to touch the camera speed in the model editor for now.

Reply:
Will do. Restoring this will also prevent mouse wheel from changing camera speed in config (which arguably I shouldn't have done anyway), so it'll primarily be handled in the upcoming settings menu.